### PR TITLE
Add tools WG

### PR DESCRIPTION
**Pull request by mgielda**
This should be an OK description to start with. It's similar to analog WG since objectives are similar. @kgugala @brianwarner @robmains @hcook please review and merge.

### CODE REVIEW

**Review by kgugala — APPROVED** [review]
LGTM

**Comment by kgugala** [comment]
@martin-lueker please take a look at this

File: working_groups/tools/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+## Mission
+
+The mission of the Tools WG will be to:
+
+* identify open source tools in the ASIC / FPGA domain and help them collaborate and interoperate
+* share and co-develop new, open source based ASIC / FPGA design methodologies
+* discuss and share experiences related to digital ASIC / FPGA design to identify gaps in the existing ecosystem and potential improvements
+* identify neighboring areas where open source ASIC / FPGA design methodologies could result in radical improvements (e.g. chiplets, memory designs etc.)
+* identify ways to use open source tools in production-level, marketable solutions, raise awareness of success stories
```
Comment (martin-lueker) [inline working_groups/tools/README.md:13]: Supremely minor suggestion:
```suggestion
* identify ways to use open source tools in production-level, marketable solutions, and raise awareness of success stories
```

File: working_groups/tools/README.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Tools Working Group
+
+The Tools working group of CHIPS Alliance covers the wide array of open source tooling for ASIC and FPGA design, mostly focusing around digital design (as there is a separate Analog WG that focuses around A/MS).
```
Comment (martin-lueker) [inline working_groups/tools/README.md:3]: Is the slash in "A/MS" commonly used?
```suggestion
The Tools working group (WG) of CHIPS Alliance covers a wide array of open source tooling for ASIC and FPGA design, mostly focusing around digital design (as there is a separate Analog WG that focuses on AMS design flows). 
```

File: working_groups/tools/README.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+The Tools working group of CHIPS Alliance covers the wide array of open source tooling for ASIC and FPGA design, mostly focusing around digital design (as there is a separate Analog WG that focuses around A/MS). 
+
+## Mission
+
+The mission of the Tools WG will be to:
+
+* identify open source tools in the ASIC / FPGA domain and help them collaborate and interoperate
+* share and co-develop new, open source based ASIC / FPGA design methodologies
+* discuss and share experiences related to digital ASIC / FPGA design to identify gaps in the existing ecosystem and potential improvements
+* identify neighboring areas where open source ASIC / FPGA design methodologies could result in radical improvements (e.g. chiplets, memory designs etc.)
```
Comment (martin-lueker) [inline working_groups/tools/README.md:12]: Minor clarification:
```suggestion
* identify neighboring areas where the application of open source ASIC / FPGA design methodologies could result in radical improvements (e.g. chiplets, memory designs etc.)
```

File: working_groups/tools/README.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
+# Tools Working Group
+
+The Tools working group of CHIPS Alliance covers the wide array of open source tooling for ASIC and FPGA design, mostly focusing around digital design (as there is a separate Analog WG that focuses around A/MS). 
+
+## Mission
+
+The mission of the Tools WG will be to:
```
Comment (martin-lueker) [inline working_groups/tools/README.md:7]: This section, the heart of the document, does a good job of covering our goals.  I'm trying to think it there is anything else we should be trying to achieve, but I think this covers our activities pretty well!  Nice job.

**Review by martin-lueker — APPROVED** [review]
LGTM.  Thanks Michael.

File: working_groups/tools/README.md (hunk 4 above)
Comment (mgielda) [reply working_groups/tools/README.md:7]: Thanks!

**Comment by mgielda** [comment]
Thanks @martin-lueker for the review. This is great. So I guess now we have to wait until the PR is 72h old :)

**Comment by mgielda** [comment]
Actually @brianwarner could we automate that? I am sure that we could have a bot that checks and does not allow to merge any PRs that are not marked as "small fix" or something. This would be pretty awesome.

**Review by hcook — APPROVED** [review]

**Comment by mgielda** [comment]
OK, we got the necessary approvals, merging.

**Pull request merged**